public/static/sdk/jsmr-0.1.js

```diff
@@ -197,30 +339,15 @@ var JsMr = Class.extend({
      * only if not it's not already in the global scope and will register the
      * client with the server.
      */
-    loadAndRegister: function() {
+    loadAndRegister: function () {
         var _this = this;
+        this.getScript(this.aws_sdk); //loads the aws sdk
```

Comment:
Loading AWS and jQuery should also be chained, since currently we call `this.register();` after jQuery is loaded which doesn't mean AWS is also loaded. so we can get into error when running the first task.